IDEAS/Buildings/Components/Examples/HorizontalFinExample.mo

```diff
@@ -63,8 +63,8 @@ public
     "Horizontal fin model with control input for displacement"
     annotation (Placement(transformation(extent={{64,20},{74,40}})));
   Modelica.Blocks.Sources.Sine sine(
-    amplitude=Modelica.Constants.pi/4,
-    offset=Modelica.Constants.pi/4,
```

Comment:
We should clearly mention that this is the reason why the reference results changed: `beta` used to go from 0 to pi/2, but by changing to the `Ctrl` input, `beta` is equal to `Ctrl*beta_max`, and `beta_max<pi/2`.

Reply:
Mentioned here: https://github.com/open-ideas/IDEAS/issues/1290#issuecomment-2210442573